main.go

```diff
@@ -34,14 +34,15 @@ func main() {
 // start adds all the commands and connects the bot to Discord.
 // Listens for CTRL+C then terminates the connection.
 func start(d *discord.Discord) {
-	d.Session.AddHandler(d.AddCommands)
+	d.Ready.Add(1)
```

Comment:
what does `1` represent?

Reply:
WaitGroup is Golang's implementation of a semaphore so it just increments the semaphore counter by 1

Reply:
ah gotcha thanks 😊